text/2020-02-04-auth.md

```diff
@@ -0,0 +1,199 @@
+# Auth for applications
+
+## Summary
+
+TiKV authentication and authorization can be implemented by leveraging
+
+* key space registration
+* token signing
+* delegation
+
+PD maintains key spaces and signs tokens for clients that authenticate.
+TiKV verifies the tokens.
+Clients may delegate their tokens to permit direct TiKV access for users in their own auth system.
+
+This can solve the most pressing auth concerns without attempting to
+
+* push down TiDB's auth into TiKV
+* develop a separate RBAC in TIKV
+
+
+## Motivation
+
+TiKV has no authentication.
+TiDB supports authentication through TiDB.
+This has been done in the TiDB layer probably in partly because it is a large effort to achieve MySQL compatible auth mechanisms.
+However, TiKV exposes its data over a network connection, whereas MySQL does not.
+
+From a security perspective, this means that any attacker that can achieve network access to a TiKV node can exfiltrate all its data.
```

Comment:
We supports SSL connections, which can limit access to authorized peers.

Reply:
Thanks, I updated the RFC to point that out.